docker/deploy.sh

```diff
@@ -17,4 +17,11 @@ echo "Pushing image";
 docker push $REGISTRY_URI:$TRAVIS_BRANCH
 
 echo "The image has been pushed";
-rm -rf .ssh/*
\ No newline at end of file
+rm -rf .ssh/*
+
+# Notifying webhook
+curl -s --output /dev/null --write-out "%{http_code}" \
```

Comment:
Check if the env are defined just in case the $AUTODEPOLY_URL does not exist or it is not setup:
```
if [ -n "$TRAVIS_TAG" ] && [ -n "$PROD_DEPLOYMENT_HOOK_TOKEN" ] && [ -n "$PROD_DEPLOYMENT_HOOK_URL" ]; then
  curl ...
fi
``